<!-- odometer.launch -->
<launch>
    <node name="odometer" pkg="igvc_navigation" type="odometer" output="screen">
    <param name="wheel_sep" type="double" value="0.556"/>
    </node>
</launch>
